avocado/utils/linux_modules.py

```diff
@@ -153,3 +159,42 @@ def module_is_loaded(module_name):
 def get_loaded_modules():
     lsmod_output = process.system_output('/sbin/lsmod').splitlines()[1:]
     return [line.split(None, 1)[0] for line in lsmod_output]
+
+
+class Config_check_mesg(Enum):
```

Comment:
Oh, sorry, I didn't mean the C-like enum, that's not really necessary. What is quite common in python is to define certain values as global names. You can see it for example in `avocado.core.exit_codes`, or basically in any bigger python module like `os.O_CREATE`.

So what I would suggest is to define after imports:

```python
#: Config commented out or not set
NOT_SET = 0
#: Config compiled as loadable module (`=m`)
MODULE = 1
#: Config built-in to kernel (`=y`)
BUILTIN = 2
```

Then in the code you'd: `return MODULE` and in tests:

```python
import linux_modules
...
conf = linux_modules.check_kernel_config(config)
if conf == linux_modules.MODULE:
    ...

PS: The travis fails because `enum` is non-core library, which is not installed by default, nor specified in ours `requirements.txt`. It's possible to add it to our dependencies, but in this case I'd recommend listing it by variables.
```